posts/maxistahl/02.server-events.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Backend to Frontend communication with Server-Sent Events
+published: true
+description: Server-Sent Events (SSE) is a technology that allows a web page to get updates from a server. In this article, I will show you how to use SSE to communicate between the backend and the frontend.
+tags: 'node.js, server-sent events, backend, frontend, web development, javascript'
```

Comment:
AFAIK you can only include 4 tags, and the format doesn't allow dots or spaces I believe.